### PR TITLE
Display deleted messages in MessageVC.

### PR DESCRIPTION
**Pull request by aforge**

### CODE REVIEW

File: Tinodios/MessageBubbleDecorator.swift [hunk 1]
```diff
@@ -102,6 +102,30 @@ class MessageBubbleDecorator {
         }
     }
 
+    public static func drawDeleted(_ rect: CGRect) -> UIBezierPath {
```
Comment (or-else) [inline Tinodios/MessageBubbleDecorator.swift]: Please use `UIBezierPath(roundedRect:,cornerRadius:)` instead:
```
return UIBezierPath(roundedRect: CGRect(x: 0, y: 0, width: size.width, height: size.height), cornerRadius: 7.61)
```

File: Tinodios/MessageViewController.swift [hunk 2]
```diff
@@ -635,9 +636,10 @@ extension MessageViewController: UICollectionViewDataSource {
     private func configureCell(cell: MessageCell, with message: Message, at indexPath: IndexPath) {
 
         cell.seqId = message.seqId
+        cell.isDeleted = message.isDeleted
 
         cell.content.backgroundColor = nil
-        if isFromCurrentSender(message: message) {
+        if message.isDeleted || isFromCurrentSender(message: message) {
```
Comment (or-else) [inline Tinodios/MessageViewController.swift]: Make the bubble color different, like in Android and web?

File: Tinodios/MessageBubbleDecorator.swift (hunk 1 above)
Comment (aforge) [reply Tinodios/MessageBubbleDecorator.swift]: Nice!

File: Tinodios/MessageViewController.swift (hunk 2 above)
Comment (aforge) [reply Tinodios/MessageViewController.swift]: Done.

**Pull request merged**